charts/incubator/channels-dvr/questions.yaml

```diff
@@ -4,6 +4,17 @@ portals:
 # Include{portalLink}
 questions:
 # Include{global}
+  - variable: imageSelector
+    group: Container Image
+    label: Select Image
+    schema:
+      type: string
+      default: image
+      enum:
+        - value: image
+          description: Latest
```

Comment:
```suggestion
          description: Channels DVR
```

Reply:
latest is very clear description I think

Reply:
Done